### PR TITLE
Feature/smb mount

### PR DESCRIPTION
**Pull request by sknorr**
### Description
Came in via mail, no bsc#. RN via F#327578 / F#327579.

### Checklist
* Check all items that apply.

*Are backports required?*

- [ ] To maintenance/SLE12SP3
- [ ] To maintenance/SLE12SP4
- [x] To maintenance/SLE15SP0

### CODE REVIEW

File: xml/net_samba.xml [hunk 1]
```diff
@@ -148,7 +148,15 @@
    </varlistentry>
   </variablelist>
  </sect1>
- <sect1 xml:id="sec.samba.install">
+ <sect1 xml:id="sec.samba.server-sled" os="sled">
+  <title>Installing a Samba Server</title>
+  <para>
+   Installing a Samba server is not supported on &sled;.
+   For information about installing and configuring a Samba server, see
+   &admin; for &sls;.
```
Comment (taroth21) [inline xml/net_samba.xml]: 'see &admin;' -> 'see the &admin;' ?

File: xml/net_samba.xml [hunk 2]
```diff
@@ -778,13 +730,74 @@
     configuration.
    </para>
   </sect2>
+  <sect2 xml:id="sec.samba.client.old-server">
+   <title>Enabling Clients to Work With SMB1 Servers</title>
+   <para>
+    The first version of the SMB network protocol, SMB1, is an old and
+    insecure protocol which has been deprecated by its originator, Microsoft.
+    For security reasons, &productname; now expects shares to use the
+    protocol version SMB 2.1, 3.0, or 3.02 by default.
```
Comment (taroth21) [inline xml/net_samba.xml]: maybe rephrase to: 'one of the following protocol versions by default: SMB 2.1, 3.0, or 3.02.'

File: xml/net_samba.xml [hunk 3]
```diff
@@ -778,13 +730,74 @@
...
+   </note>
+   <para>
+    In an existing setup, this can lead to issues in the following cases:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      If you are using an SMB server that does not support newer SMB protocol
+      versions. Windows has offered SMB 2.1 support since Windows 7 and
+      Windows Server
```
Comment (taroth21) [inline xml/net_samba.xml]: Full stop missing at the end.

File: xml/net_samba.xml [hunk 4]
```diff
@@ -778,13 +730,74 @@
...
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      If you are reliant on SMB's Unix extensions which have not been ported
```
Comment (taroth21) [inline xml/net_samba.xml]: 'If you are reliant' -> 'If you rely on'

File: xml/net_samba.xml [hunk 5]
```diff
@@ -778,13 +730,74 @@
...
+      to newer protocol versions.
+     </para>
+    </listitem>
+   </itemizedlist>
+   <important>
+    <title>Following the Instruction Below Decreases System Security</title>
```
Comment (taroth21) [inline xml/net_samba.xml]: I would prefer a shorter title: 'Decreased System Security'

File: xml/net_samba.xml [hunk 6]
```diff
@@ -778,13 +730,74 @@
...
+    <para>
+     SMB1 is still supported by the Samba server software shipped with
+     <!-- NB: profiling below is intentional - we don't support using SLED as a
+     Samba server. -->
+     <phrase os="sles;sled">&sls;</phrase><phrase os="osuse">&osuse;</phrase>.
+     SMB1 is also still supported by <command>smbclient</command>.
+    </para>
+   </note>
+   <para>
+    In an existing setup, this can lead to issues in the following cases:
```
Comment (taroth21) [inline xml/net_samba.xml]: The question is: 'which issues'? A brief high-level description of the _matter_ of issues to be expected would be helpful IMHO.

File: xml/net_samba.xml [hunk 7]
```diff
@@ -778,13 +730,74 @@
...
+     </para>
+    </listitem>
+   </itemizedlist>
+   <important>
+    <title>Following the Instruction Below Decreases System Security</title>
+    <para>
+     Following the instruction makes it possible to exploit security issues.
+     For more information, see
+     <link xlink:href="https://blogs.technet.microsoft.com/filecab/2016/09/16/stop-using-smb1/"/>.
+     Upgrade your server to allow for a more secure SMB version as soon as
```
Comment (taroth21) [inline xml/net_samba.xml]: Could we add to use the following only as a workaround or a temporary solution? Maybe like this 'Following the instruction below [...]. Use this only as a temporary workaround and upgrade your server to a more secure SMB version as soon as possible.'

File: xml/net_samba.xml [hunk 8]
```diff
@@ -778,13 +730,74 @@
...
+     possible.
+    </para>
+   </important>
+   <para>
+    If you run into either issue, you can enable support for SMB1 shares even
```
Comment (taroth21) [inline xml/net_samba.xml]: Again, which issues are we talking about here? And I would remove the word 'even'.

File: xml/net_samba.xml [hunk 9]
```diff
@@ -778,13 +730,74 @@
...
+    <option>vers=1.0</option> to the <command>mount</command> command line.
+   </para>
+  </sect2>
+  <sect2 xml:id="sec.samba.client.new-server">
+   <title>Enabling Newer Versions of the SMB Protocol on Clients</title>
+   <para>
+    To configure Samba to accept SMB 2.1, in the configuration file
+    <filename>/etc/samba/smb.conf</filename>, set the global parameter
+    <literal>server max protocol = SMB2_10</literal> (for a list of possible
+    values, see <command>man smb.conf</command>).
```
Comment (taroth21) [inline xml/net_samba.xml]: '(for a list of possible  values, see <command>man smb.conf</command>)' -> make this a separate sentence without parenthesis

File: xml/net_samba.xml [hunk 10]
```diff
@@ -1369,24 +1382,52 @@ No shadow copies found in system.</screen>
...
-   configuration.
-  </para>
+  <itemizedlist>
+   <listitem>
+    <formalpara>
+     <title>Man Pages:</title>
+     <para>
+      To see a list of all man pages installed with the package
+      <package>samba</package>, run <command>apropos samba</command>.
+      You can then open any of the man pages with
```
Comment (taroth21) [inline xml/net_samba.xml]: Open any of the man pages with: [...]

File: xml/net_samba.xml [hunk 11]
```diff
@@ -1369,24 +1382,52 @@ No shadow copies found in system.</screen>
...
+    <formalpara>
+     <title>Additional Packaged Documentation:</title>
+     <para>
+      Install the package <systemitem>samba-doc</systemitem> with
+      <command>zypper install samba-doc</command>.
+     </para>
+    </formalpara>
+    <para>
+     This documentation installs into
+     <filename>/usr/share/doc/packages/samba</filename> and contains an HTML
```
Comment (taroth21) [inline xml/net_samba.xml]: split into 2 sentences: 'It consists of an HTML version of the man pages and a library of [....]'

File: xml/net_samba.xml [hunk 12]
```diff
@@ -778,13 +730,74 @@
...
   </sect2>
+  <sect2 xml:id="sec.samba.client.old-server">
+   <title>Enabling Clients to Work With SMB1 Servers</title>
+   <para>
+    The first version of the SMB network protocol, SMB1, is an old and
+    insecure protocol which has been deprecated by its originator, Microsoft.
+    For security reasons, &productname; now expects shares to use the
+    protocol version SMB 2.1, 3.0, or 3.02 by default.
+   </para>
+   <note>
```
Comment (taroth21) [inline xml/net_samba.xml]: First saying it is deprecated and then having a note saying it is still supported in some cases come a bit as a surprise... maybe you could replace 'supported' by s.th. else?

File: xml/net_samba.xml (hunk 2 above)
Comment (sknorr) [reply xml/net_samba.xml]: I rephrased a bit differently...

File: xml/net_samba.xml (hunk 1 above)
Comment (sknorr) [reply xml/net_samba.xml]: Ignored for now. If we omit "the" here, it is easier to change the reference later.

**Pull request merged**